client/my-sites/themes/single-site-wpcom.jsx

```diff
@@ -19,17 +27,47 @@ const ConnectedSingleSiteWpcom = connectOptions( ( props ) => {
 	const upsellUrl = `/plans/${ siteSlug }`;
 	let upsellBanner = null;
 	if ( displayUpsellBanner ) {
-		upsellBanner = (
-			<UpsellNudge
-				className="themes__showcase-banner"
-				event="calypso_themes_list_install_themes"
-				feature={ FEATURE_UPLOAD_THEMES }
-				plan={ PLAN_BUSINESS }
-				title={ translate( 'Upload your own themes with our Business and eCommerce plans!' ) }
-				forceHref={ true }
-				showIcon={ true }
-			/>
-		);
+		if ( isEnabled( 'themes/premium' ) ) {
+			if ( [ PLAN_PERSONAL, PLAN_FREE ].includes( currentPlan.productSlug ) ) {
+				upsellBanner = (
+					<UpsellNudge
+						className="themes__showcase-banner"
+						event="calypso_themes_list_premium_themes"
+						feature={ FEATURE_PREMIUM_THEMES }
+						plan={ PLAN_PREMIUM }
+						title={ translate( 'Unlock ALL premium themes with our Premium and Business plans!' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Upload your own themes with our Premium and Business plans!' )` *ES Score: 15*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/themes-premium-ref-to-themes-upsell)